### PR TITLE
[master] Remove Disease Matcher

### PR DESCRIPTION
**Pull request by tommydeboer**
End of an era...

### CODE REVIEW

**Comment by dennishendriksen** [comment]
I can not allow removal of 'THE' disease matcher.

**Comment by tommydeboer** [comment]
I will not allow disallowance.

File: molgenis-data-migrate/src/main/java/org/molgenis/migrate/version/v1_13/Step22RemoveDiseaseMatcher.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+		// remove disease matcher entities + attributes
+		template.execute("DROP TABLE IF EXISTS Disease;");
+		template.execute("DROP TABLE IF EXISTS DiseaseMapping;");
+		template.execute(
+				"DELETE FROM attributes WHERE identifier IN (SELECT attributes FROM entities_attributes WHERE fullName = 'Disease' OR fullName = 'DiseaseMapping')");
+		template.execute("DELETE FROM entities WHERE fullName = 'Disease' OR fullName = 'DiseaseMapping'");
+
+		// remove settings
+		template.execute("ALTER TABLE settings_dataexplorer DROP COLUMN mod_diseasematcher");
+		template.execute("DELETE FROM attributes WHERE name = 'mod_diseasematcher'");
```
Comment (dennishendriksen) [inline molgenis-data-migrate/src/main/java/org/molgenis/migrate/version/v1_13/Step22RemoveDiseaseMatcher.java]: Does not remove entity class permissions on Disease or DiseaseMapping.

**Comment by dennishendriksen** [comment]
Please remove private static final boolean DEFAULT_MOD_DISEASE_MATCHER = true; in DataExplorerController

**Pull request merged**